### DISCUSSION TITLE
Use Other LDAP Server due  current LDAP Install failure (openldap-servers package was removed in RHEL8)

### DISCUSSION THREAD

**Discussion by agorosti** [HCL-TECH-SOFTWARE/connections-automation#89] [Ideas]
I had a failure when I was running the QuickStart ansible script because the openldap-servers package was removed in RHEL8, and was not avaible via yum

I would suggest to use Domino 11 as LDAP Server.    Domino install is easy to do,  and is an HCL product.
Additionaly,  it has a roadmap to be integrated on Connections  (Volt features) and it would be of great help if you need to deploay a fast web app for managing guest users.

**Comment by michallawson** [comment]
It would certainly make life a lot easier if the scripts used a Domino for the LDAP.
